community/portworx/templates/portworx-ds.yaml

```
@@ -170,7 +170,9 @@ spec:
               {{ end -}}
 
               "-x", "kubernetes",
-              "-metadata", "{{ printf "size=%d" (int64 $metadataSize) }}"
+              {{- if ne $metadataSize 0 }}
+              "-metadata" "{{ printf "size=%d" (int64 $metadataSize) }}",
+              {{- end -}}
```

Might be better to drop the last `-` so that the square bracket on the next line is lined up correctly in the rendered template.
```suggestion
              {{- end }}
```